packages/manager/apps/pci-gateway/src/api/hooks/useGateway.ts

```diff
@@ -9,22 +9,25 @@ import {
   getAllAggregatedGateway,
   paginateResults,
 } from '@/api/data/gateway';
-import { Gateway, GatewayResponse } from '@/interface';
+import { GatewayResponse } from '@/interface';
 
 export const useAllAggregatedGateway = (projectId: string) =>
   useQuery({
     queryKey: ['project', projectId, 'gateway'],
     queryFn: () => getAllAggregatedGateway(projectId),
     retry: false,
-    select: (data): Gateway[] => [
-      ...data.resources.map((row) => ({
-        ...row,
-        model: row.model.toUpperCase(),
-        connectedNetworkCount: uniqBy(row.interfaces, 'subnetId').length,
-        formattedIps:
-          row.externalInformation.ips.map((ip) => ip.ip).join(', ') || '',
-      })),
-    ],
+    select: (data) =>
+      data.map((row) => {
+        const formattedIps =
+          row.externalInformation.ips.map((ip) => ip.ip).join(', ') || '';
```

Comment:
```suggestion
          row.externalInformation?.ips?.map((ip) => ip.ip).join(', ') || '';
```
can this be undefined ? (not sure about it)

Reply:
It can't be undefined, for me it's doesn't necessary to add optional chaining